template.go

```diff
@@ -22,18 +22,28 @@ package main
 
 import (
 	"io/ioutil"
+	"path/filepath"
 	"time"
 
 	"gopkg.in/yaml.v2"
 )
 
 type template struct {
-	Service string            `yaml:"service"`
-	Thrift  string            `yaml:"thrift"`
-	Method  string            `yaml:"method"`
-	Headers map[string]string `yaml:"headers"`
-	Request interface{}       `yaml:"request"`
-	Timeout time.Duration     `yaml:"timeout"`
+	Peers           []string          `yaml:"peers"`
+	Peer            string            `yaml:"peer"`
+	PeerList        string            `yaml:"peerList"`
+	Caller          string            `yaml:"caller"`
+	Service         string            `yaml:"service"`
+	Thrift          string            `yaml:"thrift"`
+	Procedure       string            `yaml:"procedure"`
```

Comment:
Why the rename? The option is currently named method everywhere else in yab.

If we want to change it to procedure, we should do that everywhere separately, and support `method` as an alias for `procedure` for backwards compatibility in flags.